kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/model/OrderConfirmation.java

```diff
@@ -0,0 +1,26 @@
+package com.kin.ecosystem.data.model;
+
+import com.kin.ecosystem.network.model.Order.Status;
```

Comment:
this is network layer object - it should be abstracted from app related model object (even if identical now) this will allow us to change network objects without affecting the app developer - using adapter approach that translate between network object to app related model object